### PR TITLE
Update daily-deploy-schedule.md

### PR DESCRIPTION
**Pull request by benjaminchait**
inspired by @jskinne3 during retro

### CODE REVIEW

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/daily-deploy-schedule.md [hunk 1]
```diff
@@ -47,3 +47,7 @@ Deploys are scheduled every 6 hours at **5am, 11am, 5pm, 11pm UTC**.
 | 11am | <lg-local-time utc="11am" />   |
 | 5pm  | <lg-local-time utc="5pm" />    |
 | 11pm | <lg-local-time utc="11pm" />   |
+
+### Environment status
+
+You can view the [Envrionment status](https://dashboard.int.identitysandbox.gov/env) which includes the SHA and the date of last deploy.
```
Comment (aduth) [inline _articles/daily-deploy-schedule.md:53]: Small typo on "Environment"

```suggestion
You can view the [Environment status](https://dashboard.int.identitysandbox.gov/env) which includes the SHA and the date of last deploy.
```

**Review by jskinne3 — APPROVED** [review]
Thank you!

**Comment by benjaminchait** [comment]
@aduth - Good eye!

**Pull request merged**